src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base.html.twig

```diff
@@ -21,8 +21,8 @@
                 document.body.classList.add(localStorage.getItem('symfony/profiler/theme'));
             }
             // needed to respond dynamically to OS changes without having to refresh the page
-            window.matchMedia('(prefers-color-scheme: dark)').addListener(function (e) {
-                document.body.classList.remove(e.matches ? 'theme-light' : 'theme-dark');
+            window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', () => {
```

Comment:
@javiereguiluz shouldn't this listener be registered only when the auto theme is used ?

Reply:
Yes! You are right. I've made some changes. Thanks!